src/main/java/de/tum/in/www1/artemis/domain/Achievement.java

```diff
@@ -37,6 +37,12 @@
     @Column(name = "achievement_type")
     private AchievementType type;
 
+    @Column(name = "parameter")
+    private Long parameter;
```

Comment:
What does `parameter` mean? Is that documented somewhere? Or can you maybe use a more describing name?

Reply:
The `parameter` is the property of an achievement which is used to define whether someone earned an achievement when checking for fulfilment. For example, for a point-based achievement the score needed for a golden achievement can be set to 90 or 100, for silver to 80 and for bronze to 70 etc.
If it is a time-based achievement, the number of days within which the submission must occur to receive an achievement can be set. These thresholds are stored in the `parameter`. 
So maybe `thresholdValue` would be more appropriate and describing here. What do you think?

Reply:
Yes, something like `thresholdValue` (maybe just `threshold`) or `successCriteria` would fit better here in my opinion.